### PR TITLE
Add Feature Projects List and Info About my self

### PR DESCRIPTION
**Pull request by Mar12358**
**Need Feedback to be able to continue:**
- **I went through a rabbit hole and spent many hours** trying to resize using flex, when I add items to the container they expand beyond the container's boundary. so I decided to reach out for help.

-  When finishing one card I'll use it as a template for the other ones.
 - I've put borders to easily see the sizes of each element. 

- [ ] **I'm not being able to properly adjust sizes, I'm trying with justify-content: space-around. But for some reason, the spaces between the items in the "card_bottom" container don't want to shrink.**

- Please, I need help
[Here is an image of the card](https://github.com/Mar12358/Portfolio/blob/feature-list-project/Issue1.jpg)
[Here is an image of the "About my self" Section](https://github.com/Mar12358/Portfolio/blob/feature-list-project/Issue2.jpg)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -39,7 +41,88 @@ <h1 class="salute">I'm Alexandra. Glad to see you!</h1>
...
+        <section id="stories2">
+            <div class="card">
+                stories2
+            </div>
+        </section>
+    </div>    
+    <section class="about_my_self">
+        
+        <div class="content content_about_self">
+            <h1 class="salute">About Myself</h1>
```
Comment (mahbubul14) [inline index.html]: - [ ] Kindly change this `h1` to other `hx` tag. It is better for your page to have just one h1 tag, this makes your website semantic and easily readable by browsers 👍🏿

File: index.html [hunk 2]
```diff
@@ -10,14 +10,16 @@
...
         <header class="app-header">
             <a class="logo" href="#">My Logo</a>
             <nav class="nav-bar">
                 <img class="menu" src="Union.png" alt="menu">
             </nav>
         </header>
-        <div class="content">
+        <div class="content welcome_content">
             <h1 class="salute">I'm Alexandra. Glad to see you!</h1>
             <p class ="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
```
Comment (mahbubul14) [inline index.html:24]: - [ ] Kindly remove the border from the outside of the text to match the Figma design. :+1: 
remove_border

File: index.html [hunk 3]
```diff
@@ -39,7 +41,88 @@ <h1 class="salute">I'm Alexandra. Glad to see you!</h1>
...
+                        <li><p class="text_li">2015</p></li>
+                    </ul>
+                    <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="categories">
+                        <li><p>HTML</p></li>
+                        <li><p>CSS</p></li>
+                        <li><p>JavaScript</p></li>
+                    </ul>
+                    <a class="see_project_button" href="#">See project</a>
+                </div>
```
Comment (mahbubul14) [inline index.html]: - [ ] Kindly remove the border color and try to match the Figma design.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
yourdessign_card | figma_card

File: index.html [hunk 4]
```diff
@@ -39,7 +41,88 @@ <h1 class="salute">I'm Alexandra. Glad to see you!</h1>
...
+        <section id="tonic2">
+            <div class="card">
+                tonic2
+            </div>
+        </section>
+        <section id="stories2">
+            <div class="card">
+                stories2
+            </div>
+        </section>
```
Comment (mahbubul14) [inline index.html]: - [ ] Kindly complete all cards for the work section and try to match the Figma design. :+1:

File: index.html [hunk 5]
```diff
@@ -39,7 +41,88 @@ <h1 class="salute">I'm Alexandra. Glad to see you!</h1>
...
+            </ul>
+            <a class="get_resume_button" href="#">Get my resume</a>
+            <ul class= "content_about_self2">
+                <li class="list_info"><p>Languaje</p></li> 
+                <li class="list_lang"><p>JavaScript</p></li>
+                <li class="list_lang"><p>HTML</p></li>
+                <li class="list_lang"><p>CSS</p></li>
+                <li class="list_info"><p>frameworks</p></li>
+                <li class="list_info"><p>Skills</p></li>
+            </ul>
```
Comment (mahbubul14) [inline index.html]: - [ ] Your about myself section is not matched with the Figma design. Kindly work on it. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
yourdesign_about_myself | figma_about_myself

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @Mar12358,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Mar12358** [comment]
**Added Features:**
- Add Cards in "Work" Section.
- Complete "About Myself" Section.
- Add Style according to Figma Template.
- Apply changes According to Feedback

**Completed required checklist:**

- [ ] "About myself" section is aligned with flexbox.
- [ ] Use CSS Grid in the main container
- [ ] Use Flexbox to lay out an individual card
- [ ] Use `ul `and `li` for categories in "Work" section. `Html,` `Css,` and `Javascript` should be list items.
- [ ] use `ul` and `li` for categories in "About myself" section. `Languages`, `Frameworks`, and `Skills` should be list items.
- [ ] Stick to the design as much as possible 
- [ ] Implement the button interactions

**Review by binhussen — APPROVED** [review]
Hi @Mar12358 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**